person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] To avoid the _wrong number of arguments_ error. Please update this syntax to as follows `(name = 'Unknown', parent_permission = true)`.

Reply:
@jasonfelice  using this gives `name = 'Unknown', parent_permission = true` me the linter errors

**Style/OptionalBooleanParameter: Prefer keyword arguments for arguments with a boolean default value; use parent_permission: true instead of parent_permission = true.**

Reply:
Fixed.